backend/workflow_manager/endpoint_v2/destination.py

```
@@ -234,7 +235,7 @@ def handle_output(
                 input_file_path=input_file_path,
                 file_execution_id=file_execution_id,
             ):
-                self.insert_into_db(input_file_path=input_file_path)
+                self.insert_into_db(input_file_path=input_file_path, error=error)
```

Ruff (0.12.2)

Undefined name `error`

(F821)